### ISSUE TITLE
Enhance reading list for new team members

### ISSUE THREAD

**Issue by iteles** [ideaq/start-here#8]
What subjects are **essential** for **everyone** on our team to **[grok](http://en.wikipedia.org/wiki/Grok)**? Need to create a prioritised list.

**Comment by nelsonic** [comment]
Agree 100%.
my "***home page***" (*AKA LinkedIn* http://linkedin.com/in/nelsonic) has had a ***Recommended Reading*** **List** for a *while* but these are *non*-technical books... 
also tweeted a list of books a couple of times:
+ https://twitter.com/nelsonic/status/456759766917083136
+ https://twitter.com/nelsonic/status/560002471058497536
But so far not made a good *list* other people tick off against (and *add* to!) #**mybad** :frowning:
(sometimes I ***assume everyone already knows everything*** by osmosis ... #**fail**)

**Comment by nelsonic** [comment]
My "*Short*" List:
1. Start With Why: http://www.amazon.com/Start-Why-Leaders-Inspire-Everyone/dp/1591846447
1. Don't Make Me Think: http://www.amazon.co.uk/Dont-Make-Me-Think-Usability/dp/0321965515
2. The Happiness Advantage: http://www.amazon.com/Happiness-Advantage-Principles-Psychology-Performance/dp/0307591549
3. Delivering Happiness: http://www.amazon.com/Delivering-Happiness-Profits-Passion-Purpose/dp/0446576220
4. The Lean Startup: http://www.amazon.com/Lean-Startup-Entrepreneurs-Continuous-Innovation/dp/0307887898
5. Web Analytics: http://www.amazon.com/Web-Analytics-2-0-Accountability-Centricity/dp/0470529393
6. JavaScript The Good Parts: http://www.amazon.co.uk/JavaScript-Good-Parts-Douglas-Crockford/dp/0596517742
7. Testable JavaScript: http://www.amazon.com/Testable-JavaScript-Mark-Ethan-Trostler/dp/1449323391
8. You should test that: http://www.amazon.co.uk/You-Should-Test-That-Optimization/dp/1118301307

**Comment by iteles** [comment]
We also need a way of not overwhelming people with a reading list a mile long! :book:

**Comment by nelsonic** [comment]
#**GreatMinds** :stuck_out_tongue_winking_eye:
I've always thought it should be [*gamified*](http://en.wikipedia.org/wiki/Gamification) and people should *un-lock* levels of learning... @tonydaly might have some insight on this.

**Comment by nelsonic** [comment]
Should we move this to: https://github.com/dwyl/alc ?

**Comment by iteles** [comment]
Yes! For now it should be an md linked into the readme, which we should _absolutely_ also link to from the readme in start-here.
